### PR TITLE
Bug 1116511: Add tableau data sync and JSON view for MozID.

### PR DESCRIPTION
**Pull request by pmclanahan**
Pulls in contributor data directly from the Tableau DB. These
data are published to tableau once a week. Provides a JSON view
for the various views of the data which is set to be cached for
a week. Adds the Apache config for the new URLs. Adds cron jobs
for stage and prod to sync the data weekly. Does not yet remove
the old static files, will do that once the working copies of
the site are using the new datasource.

### CODE REVIEW

**Comment by pmclanahan** [comment]
@jgmize r?

File: scripts/update_tableau_data.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+from django.conf import settings
+
+import MySQLdb
+
+from bedrock.mozorg.models import ContributorActivity
+
+
+urlparse.uses_netloc.append('mysql')
+QUERY = ('SELECT c_date, team_name, source_name, count(*) AS total, IFNULL(SUM(is_new), 0) AS new '
+         'FROM contributor_active {where} GROUP BY c_date, team_name, source_name')
```
Comment (jgmize) [inline scripts/update_tableau_data.py]: That's some nice hand crafted, artisanal sql you have there.

**Comment by jgmize** [comment]
What is the plan for dev, demo, and staging environments? Do we have access to separate dev and/or staging Tableau DBs? Can we put this on one of the demo environments with the appropriate settings added for testing?

**Comment by pmclanahan** [comment]
I've checked out demo4 and am pushing this branch to it as I type. You should just need to temporarily add the setting I sent you earlier to `settings/local.py` and run `./manage.py runscript update_tableau_data`. You'll then be able to see data at the JSON urls:

* https://www-demo4.allizom.org/contributor-data/firefox.json
* https://www-demo4.allizom.org/contributor-data/firefoxos.json
* https://www-demo4.allizom.org/contributor-data/all.json
* etc.

**Comment by pmclanahan** [comment]
I went ahead and did this, and the above urls are now working and returning data \o/

**Comment by jgmize** [comment]
Thanks @pmclanahan, I think this just needs a rebase and then we're good to go.

**Comment by pmclanahan** [comment]
Just rebased this on latest master @jgmize.

**Pull request merged**